R/parsing_partitions.R

```diff
@@ -0,0 +1,46 @@
+#path <- getwd()
+#Must have this
+setwd('/home/striker/Desktop/UCLA/BS Research Paper/SideWork')
+
+#install.packages("stringr", repos='http://cran.us.r-project.org')
+library("stringr")
+
+#As a start, try turning the bit that parses the .partitions file into a function 
+#that returns a data frame with three columns, 
+
+#the first being the gene name, 
+#the second being the starting offset, 
+#and the third being the ending offset.
+
+parsing_partitions <- function(file_name) {
+  path <- getwd()
+  
+  fileName <- file_name
+  
+  #using readlines instead of scan and capturing the relevant text from each line using str_match
+  full_name <- paste("/", fileName, sep="")
+  readLinesText <- readLines(paste(path, full_name, sep = ""))
```

Comment:
You shouldn't need to create paths inside of a function; instead, this should be the caller's responsibility to appropriately construct the correct path. (Also, you should be using `file.path` instead of `paste` to safely construct paths).